api/turing/utils/kubernetes.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+func pad(s string, rightPaddingLength int) string {
+	var sb strings.Builder
+	for i := 0; i < rightPaddingLength; i++ {
+		sb.WriteString("0")
+	}
+
+	return fmt.Sprintf("%s%s", s, sb.String())
+}
+
+// IsQualifiedKubernetesName allows for padding of name before checking if it is DNS qualified
+func IsQualifiedKubernetesName(s string, rightPaddingLength int) bool {
+	paddedString := pad(s, rightPaddingLength)
```

Comment:
The padded value doesn't matter, the only thing that matters is that there is some compliant value there for testing that it would work. 

Here I append the timestamp string so I need to make sure that 11 characters is reserved for it.
https://github.com/ashwinath/turing/blob/fix-ensembler-orchestration/api/turing/service/ensembling_job_service.go#L139-L142

Reply:
Ok, I got the logic now, but it seems contra-intuitive to me. Why wouldn't you just use `go-playground/validator` to validate the ensembler's name max length + validate that it is qualified kubernetes name?

Reply:
Ya thinking about it that makes sense too. The padding is not required.

Reply:
I removed the padding.